yarn-project/txe/src/txe_service/txe_service.ts

```diff
@@ -157,9 +130,10 @@ export class TXEService {
     return toForeignCallResult([toArray(publicDataWrites.map(write => write.value))]);
   }
 
-  async createAccount() {
+  async createAccount(secret: ForeignCallSingle) {
     const keyStore = (this.typedOracle as TXE).getKeyStore();
-    const completeAddress = await keyStore.createAccount();
+    const secretFr = fromSingle(secret);
+    const completeAddress = await keyStore.addAccount(secretFr, secretFr);
```

Comment:
@Thunkar why is it ok for the `partialAddress` (arg 2 of this `addAccount` function call) to be a dummy value of `secretFr` instead of an actual `partialAddress`?

Reply:
Because this fn doesn't really create an account that gets deployed anywhere (and thus, no contract). It's just a dummy that has some keys and allows TXE to do its thing (as long as you don't have to interact with the account "on-chain", such as dealing with authwits!)